iterators.tex

```diff
@@ -347,7 +347,7 @@
   template <class Out, class T>
   concept bool MoveWritable@\newtxt{() \{}\oldtxt{ =}@
     @\newtxt{return}@ Semiregular<Out>@\newtxt{()}@ &&
-      requires (Out o, T v) {
+      requires (@\newtxt{const }@Out o, T v) {
         *o = move(v); @\newtxt{// not equality preserving}@
```

Comment:
*This* is definitely a problem. I had a hunch we'd hit on something that modifies on assignment. I'll back it out here and in Writable..